reflect/src/main/java/dagger/reflect/ReflectiveModuleParser.java

```diff
@@ -29,11 +32,13 @@ static void parse(Class<?> moduleClass, @Nullable Object instance,
         }
 
         Binding<?> binding;
+        TypeWrapper wrapper = TypeWrapper.NONE;
```

Comment:
is it possible to combine @BindsOptionalOf + @IntoSet? we'll probably cross that bridge when we get there